packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.Input/src/InputTypes/InputParameter.cs

```diff
@@ -20,7 +20,6 @@ public InputParameter(
             InputOperationParameterKind kind,
             bool isRequired,
             bool isApiVersion,
-            bool isResourceParameter,
```

Comment:
Why is this being removed?

Reply:
this is never used anywhere just for swagger backward compatibility.
If in the future we need this, we should add this flag via decorator values like we did for the resources in the new mgmt plugin.